src/nupic/frameworks/opf/htm_prediction_model.py

```diff
@@ -69,6 +70,7 @@ def requireAnomalyModel(func):
   """
   Decorator for functions that require anomaly models.
   """
+  @wraps(func)
```

Comment:
Without this, we lose the original docstrings on introspection, and Sphinx cannot get to them.